### PR TITLE
Sort instructions from newest to oldest versions

### PR DESCRIPTION
**Pull request by Tbohunek**
# PR Summary

I suggest to sort steps from newest to oldest. Gotta lead by example and promote latest versions. 😉 

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [x] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [ ] Preview content
- [ ] Version 7.1 content
- [ ] Version 7.0 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

**Comment by sdwheeler** [comment]
@SteveL-MSFT Can we get someone to review these changes?

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 1]
```diff
@@ -29,33 +29,28 @@ the OS in separate steps.
...
 Officially supported platform releases for PowerShell 7.1
 
-- Ubuntu 16.04/18.04/20.04 (including ARM64)
-- Ubuntu 19.10 (via Snap package)
-- Debian 9/10
-- CentOS and RHEL 7/8
-- Fedora 30
-- Alpine 3.11+ (including ARM64)
+- Ubuntu `20.04`/`18.04`/`16.04` (including ARM64)
+- Ubuntu `19.10` (via Snap package)
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md:33]:
```suggestion
- Ubuntu `19.10` (via Snap package)
```
This is not officially supported.

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 2]
```diff
@@ -29,33 +29,28 @@ the OS in separate steps.
...
-- openSUSE 42.3
-- openSUSE Leap 15
+- Ubuntu `20.04`/`18.04`/`16.04`
+- Debian `10`/`9`/`8`
+- Alpine `3.10`/`3.9`
+- CentOS `7`
+- Red Hat Enterprise Linux (RHEL) `7`
+- Fedora `30`/`29`/`28`
+- openSUSE `42.3`
+- openSUSE Leap `15`
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md:49]: leap is not officially supported

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 3]
```diff
@@ -29,33 +29,28 @@ the OS in separate steps.
...
-- Fedora 30
-- openSUSE 42.3
-- openSUSE Leap 15
+- Ubuntu `20.04`/`18.04`/`16.04`
+- Debian `10`/`9`/`8`
+- Alpine `3.10`/`3.9`
+- CentOS `7`
+- Red Hat Enterprise Linux (RHEL) `7`
+- Fedora `30`/`29`/`28`
+- openSUSE `42.3`
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md:48]: This is EOL and no longer officially supported.

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 4]
```diff
@@ -29,33 +29,28 @@ the OS in separate steps.
...
-- Fedora 29
-- Fedora 30
-- openSUSE 42.3
-- openSUSE Leap 15
+- Ubuntu `20.04`/`18.04`/`16.04`
+- Debian `10`/`9`/`8`
+- Alpine `3.10`/`3.9`
+- CentOS `7`
+- Red Hat Enterprise Linux (RHEL) `7`
+- Fedora `30`/`29`/`28`
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md:47]: These are all EOF and thus, no longer officially supported.

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 5]
```diff
@@ -29,33 +29,28 @@ the OS in separate steps.
...
-- CentOS 7
-- Red Hat Enterprise Linux (RHEL) 7
-- Fedora 28
-- Fedora 29
-- Fedora 30
-- openSUSE 42.3
-- openSUSE Leap 15
+- Ubuntu `20.04`/`18.04`/`16.04`
+- Debian `10`/`9`/`8`
+- Alpine `3.10`/`3.9`
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md:44]: 3.9 is EOL and thus, no longer officially supported.

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 6]
```diff
@@ -29,33 +29,28 @@ the OS in separate steps.
...
-- Alpine 3.9 and 3.10
-- CentOS 7
-- Red Hat Enterprise Linux (RHEL) 7
-- Fedora 28
-- Fedora 29
-- Fedora 30
-- openSUSE 42.3
-- openSUSE Leap 15
+- Ubuntu `20.04`/`18.04`/`16.04`
+- Debian `10`/`9`/`8`
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md:43]: 8 is EOL and thus, no longer officially supported.

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 7]
```diff
@@ -29,33 +29,28 @@ the OS in separate steps.
...
-- Debian 9/10
-- CentOS and RHEL 7/8
-- Fedora 30
-- Alpine 3.11+ (including ARM64)
+- Ubuntu `20.04`/`18.04`/`16.04` (including ARM64)
+- Ubuntu `19.10` (via Snap package)
+- Debian `10`/`9`
+- CentOS `7`/`8`
+- RHEL `7`/`8`
+- Fedora `30`
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md:37]: 30 is EOL and thus, no longer officially supported.

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 8]
```diff
@@ -29,33 +29,28 @@ the OS in separate steps.
...
+- Alpine `3.11`+ (including ARM64)
```
Comment (adityapatwardhan) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md:38]: Alpine ARM64 is not supported yet.

**Comment by Tbohunek** [comment]
Just for the record, I only reordered the text, suggesting approach to be taken generally.
Maybe cleanup in terms of actually supported versions should be handled separately?

**Comment by sdwheeler** [comment]
@Tbohunek Thanks for the contribution but we can't take these changes as submitted.

**Comment by Tbohunek** [comment]
@sdwheeler cool, fine with me, just not sure what you mean by "can't take these changes as submitted". Did I miss anything, shall I adjust the proposal, or is this type of contribution not desired? Thanks.

**Comment by sdwheeler** [comment]
I agree that the article needs to be updated. But too many of the changes are incorrect. I suggest that you open an issue describing what you think needs to be corrected or changed. I will work with the PowerShell team to get the correct information into the article.

**Pull request closed without merging**